kraken/litmus/common_litmus.py

```diff
@@ -25,6 +25,13 @@ def run(scenarios_list, config, litmus_uninstall, wait_duration, litmus_namespac
                 if yaml_item["kind"] == "ChaosEngine":
                     engine_name = yaml_item["metadata"]["name"]
                     experiment_names = yaml_item["spec"]["experiments"]
+                    experiment_namespace = yaml_item["metadata"]["namespace"]
+                    if experiment_namespace != "litmus":
+                        logging.error(
+                            "Specified namespace in the scenario: %s is not supported, please switch it to litmus"
```

Comment:
Can you add what file the incorrect namespace is in, might be easier to find that way

Reply:
Done. PTAL.